p2p/antiflood/p2pAntiflood.go

```diff
@@ -50,6 +54,24 @@ func (af *p2pAntiflood) CanProcessMessage(message p2p.MessageP2P, fromConnectedP
 	return nil
 }
 
+// CanProcessMessageOnTopic signals if a p2p message can be processed or not for a given topic
+func (af *p2pAntiflood) CanProcessMessageOnTopic(message p2p.MessageP2P, fromConnectedPeer p2p.PeerID, topic string) error {
+	topicFloodPreventer := af.TopicFloodPreventer
+	if check.IfNil(topicFloodPreventer) {
+		return p2p.ErrNilTopicFloodPreventer
+	}
+	if message == nil {
+		return p2p.ErrNilMessage
+	}
+
+	ok := topicFloodPreventer.Accumulate(message.Peer().Pretty(), topic)
+	if !ok {
+		return fmt.Errorf("%w in p2pAntiflood for originator", p2p.ErrSystemBusy)
```

Comment:
might output the pid.Pretty also in the error

Reply:
done